src/api-service/__app__/onefuzzlib/azure/containers.py

```diff
@@ -31,6 +31,8 @@
     get_storage_account_name_key_by_name,
 )
 
+JOB_SCHEDULING_ESTIMATED_MAX_DELAY = datetime.timedelta(days=1)
```

Comment:
Actually that's worth a work item of it's own to figure out how long in worst case we wait for a job to become scheduled

Reply:
I'll just match the existing default duration that we currently have for the jobs instead. 
The current code create sas of 30 days